src/app/features/characters/pages/character/character.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { CustomMarvelChar } from '@characters/models/characters.model';
+import { CharactersMarvelService } from '@characters/services/characters-marvel.service';
+import { Subscription } from 'rxjs';
+
+@Component({
+  selector: 'character',
+  standalone: true,
+  imports: [],
+  templateUrl: './character.component.html',
+  styleUrl: './character.component.scss'
+})
+export class CharacterDetailsComponent {
```

Comment:
you also need to rename files according to the class name change

Reply:
done